data-access-analysis-tools/index-nb-intro.rst

```diff
@@ -60,7 +60,8 @@ Pressing the orange start button to start the server returns this page with a bl
 **JupyterLab Navigation:**
 The JupyterLab landing page in the figure below is the launch pad for all JupyterLab functionality (e.g., Notebook, Terminal).
 Return to this launch pad at any time by clicking the plus symbol at upper left.
-For DP0.1, only Python Notebooks and the terminal interface are supported (i.e., not the Console), and users are not able to access Portal queries from a Jupyter Notebook.
+For DP0.1, most users will prefer using the Notebook or Terminal, but there is also a Python Console available.
+In order to duplicate catalog queries executed in the Portal, re-execute the query using the TAP client available in the Notebook
```

Comment:
We can do even better than this already.  I will try to provide a demo notebook by next week.  It's very simple; it's just that it's not at the top of my DP0.1 stack.